Added initial Db Script

snipered2003 opened the pull request:
Removed create database script and added default utcdatetime

jimwashbrook commented:
I realise this isn't in the schema that was discussed, but would it be worth to set things like:

- `dateCreated` -> set on create to `GETUTCDATE()`
- `dateLastUpdated` -> same, but also on modified

etc.?

snipered2003 commented:
> I realise this isn't in the schema that was discussed, but would it be worth to set things like:
> 
> * `dateCreated` -> set on create to `GETUTCDATE()`
> * `dateLastUpdated` -> same, but also on modified
> 
> etc.?

Removed the database create script and added default UTC to the dateCreated

MartinSandersAND reviewed (approved):
Good.
